support-frontend/assets/components/onboarding/appBadgesDownload/guardianAppsQrCode.tsx

```diff
@@ -0,0 +1,31 @@
c58+BKm0Z3P58ViMZ/Pu14YmCMWljOVL034zAPex94Nd3jDwMf93OTIrYFvLybtRaVJO72mafp+rR+5yblysTNP256PgKCKnN/i+U8DP3MBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKyguAQSifMZvN+t7CR9zcz0g7vYHfpoE/5Lfp4lMdLivtHGLzatK2l+niN7HO6a3X67TTS7Ner/s+14+0bdv3CX0kbZRT27aBhXwEBBdzd3fX9xYub+Av6vbmg8bEzkEAAIoSAICiBACgKAEAKEoAAIoSAICiBACgKAEAKEoAAIoSAICiBACgKAEAKEoAAIoSAICiBAAuZjKZ9L2Fy7vJF5XmfD7nLBS7TdOL7+N/eXl5OR6P02mHFYc/IWi9Xg/229Jns9lyuex7F4Pw9PR0f3//+vr6zy+Jnd7hcBiNRp0WijmdTqvVqutVgffgZDJ5f1FdtW3b6Rvq3xd6fHwMrDVki8WiaZpOZz4ajXa7Xad/H75NeQE4Ho8/fvxIWy5H4E1Ivvv7+2/fviUslPnz6637SLXM9+DDw0POQgM3n8/n83nXq8bj8Vds5r/lfQTUtYFwKQm/kl+Fgb8H3aZ8/g8AoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAIBBiM20MrDsM6ZdR898xna7TVsrIOco0mY5heep7ff7TrOcwgLPQ+aYsyGPezudTn1v4Te6PkXhmVZpU9gCzufzYrEIDITp+uxdwTy17Xb7NmCZcQpsr2matO2lid2pwEIDf/bSDPw3sJvUNE3OzX1+fg5sz0dA12Hgs5yAv5X2zo39DSQAAEUJAEBRAgBQlAAAFCUAAEUJAEBRAgBQlAAAFCUAAEUJAEBRAgBQlAAAFCUAAEUJAEBReQEY+AiLgW+PzwjPxoFPGvgPlulbaMLG7VmtVo7i3e2dQ2yIWOAcNpvNn3/+2fWq9XodGBI3cAN/isbjcddL2rZ9eHj4is30yEdAcDGxEZKDHTzJf8qZk5pMAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQA4Pcmk0nfW7i86W6363sPtZxOp7TpH23bdv0S89gAo5eXl+PxOJ1OOy0UO4f1ep3wBfoDH+R0kwJP0fl8XiwW8/n863b1y+FwGI1Gr6+vX71Q5o+I6R9//JGzEr+kDUtKG2B0PB5//PjR9arYOdze8CzexZ6ipmlyAvD4+Jiwyru0HxE+AuICOv3WBn8r9hR59j5DAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogTgOpzP57638JGbHJYU4DZ9Ruz0Bn7mAxf8Ku3AqKnbM5vNlstlzlqLxaJpmq4jt2JrbTabTiO3JpPJ+7Ckrvb7fWBgWWAgTNrAssBtepcwmC98m9IETu99ItjXbek/PT093d/fJ0wEG0Wfh+122/WScWCZUeLAmoEbjyMHOPDTi72oNIHT+9e//pU2sCxgt9vd3mC+23vIn5+fv3379gV7+RuB7W232+/fv3e9ykdA3D5Do/i8nN/9kwkAQFECAFCUAAAUJQAARQkAQFECAFCUAAAUJQAARQkAQFECAFCUAAAUJQAARQkAQFECAFCUAHD7ZrNZ31v4SHh0D5kG/hSlbu+tu/V6nbe/7tbrdeBFxdYa+OkFtheT9orSxJ6iNG3b9n1C5WT+YAnI+wug05TBfLbH5w38Nhnjmm/gj4SPgACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogQAoCgBAChKAACKEgCAogTgOpzP5763wO+5TfzFwB+Jad8b4B9ZLBZN00ynGfdrt9slrJI5wOjp6en+/v719fVLVzmfz4vFInDhy8vL8XjsdHNPp9NqtQqsFRA4vdlstlwuExaaTCaHw+Hx8bHrWm3bJkxHCD8SAbGHXACuw3w+n8/nOWuNx+OchdLc399/+/at7138T8fj8cePH12vSpsblXZ6mbfp4eEhZ6E0sdPzERC376t/9/+knD/swtJOL7bQwG9umtg5CABAUQIAUJQAABQlAABFCQBAUQIAUJQAABQlAABFCQBAUQIAUJQAABQlAABFCQBAUQIAUJQAcPsmk0nfW7hiAz+9gY/cShO7TYP+InI+abPZ3N3ddbokc1BXmsPhMPr6L45/H/+UNrcnTeD0Mp+iwLC80+kUWCgwuC0mdno5D/m/vXW33W4zdha13W4DLyq2VmChmMseEb/VNE3gNsXeGmkLpYm9B9M0TdP3CV2ej4DgYgY+24vPuMmbKwAARQkAQFECAFCUAAAUJQAARQkAQFECAFCUAAAUJQAARQkAQFECAFCUAAAUJQAARQkAQFF5AYgNYUgz8O1xFTxFnzHw07vJWUl533C9Wq3ejCi5BgO/TePxOGeh2zuH94krXa/a7/fL5bLrVbd3ejFt2z48PHS9Km17PgICPvLz58++t3DFBn56AgBQlAAAFCUAAEUJAEBRAgBQlAAAFCUAAEUJAEBRAgBQlAAAFCUAAEUJAEBRAgBQlAAAFCUAwEcmk0nfW+CrBAfC7Pf7gX/PdYLMCUEvLy/H43E67Xa/ttttYK3dbtfp35/P58ViMZ/Puy602Wzu7u66XhV7UV0NfDrVaDRq2zbwHux6cyeTyeFw6LpKTOwhD7jJ2V5PT0/39/evr699b4TfeeuuaZqchWKDnJqmyVko/KKGLJa0tIViAtuLPeQD9z6FravAQs/Pz4GFfAR0HRJ+LfqMgW+Pq+Ap+ozY7/4CAFCUAAAUJQAARQkAQFECAFCUAAAUJQAARQkAQFECAFCUAAAUJQAARQkAQFECAFCUAAAUJQAARf0/EQp9yQV8c3sAAAAASUVORK5CYII="
```

Comment:
seems a bit random that we have an TS -> SVG -> base64 encoded PNG?  I'm not sure what best practice is but why not just import the png directly?

Reply:
this data links to `https://guardian.go.link/bJSYg` which redirects to 
`https://play.google.com/store/apps/details?id=com.guardian&referrer=adjust_reftag%3DcJuTrQAC52vt3%26utm_source%3DOn%2Bplatform%26utm_campaign%3DSupporter_Onboarding`
although the adjust_reftag varies